demo/kind-deploy/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:latest
+
+# Skip past interactive prompts during apt install
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt update && apt install libssl-dev build-essential pkg-config \
+     git gcc clang libclang-dev python3-pip hub numactl cmake \
+     curl openjdk-19-jdk maven postgresql-client postgresql netcat jq -y 
+
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+
+RUN git clone --branch pipeline_server http://www.github.com/lalithsuresh/database-stream-processor \
+    && git clone http://www.github.com/vmware/sql-to-dbsp-compiler
+
+RUN cd database-stream-processor && ~/.cargo/bin/cargo build
+
+RUN cd sql-to-dbsp-compiler/SQL-compiler && mvn -DskipTests package
+
+# Install rpk
+RUN curl -1sLf 'https://packages.vectorized.io/nzc4ZYQK3WRGd9sy/redpanda/cfg/setup/bash.deb.sh' | bash && \
```

Comment:
I would move this and the next command up, right before cloning the DBSP repos, to speed up container rebuilds.

Reply:
Also, we probably want to add container creation to the DBSP build process, which means that we want to use local copies of DBSP and SQL compiler rather than pulling from github (I am reorganizing the repos now so that the SQL compiler will probably become a git submodule of dbsp -- still thinking of the best way to do this, but point is, you should be able to count on matching versions of both repos being available locally).